intercom-common/src/tyhandlers.rs

```diff
@@ -3,17 +3,31 @@ use std::rc::Rc;
 use syn::*;
 use quote::Tokens;
 
+use methodinfo::Direction;
+
 use ast_converters::*;
 
-/// Defines tokens for converting COM types into Rust types
-pub struct ComToRust
-{
-    /// Optional expression for storing a temporary value in the stack
-    /// for the duration of the Rust call.
-    pub stack: Option<Tokens>,
+/// Type usage context.
+pub struct TypeContext {
+    dir: Direction,
+}
```

Comment:
What type of context information can we add here in the future?

Reply:
Any context information that affects the types used but doesn't necessarily need a whole new type handler to be created. One example is the Automation types vs Raw types (BSTR vs char*)